src/helpers/datetime.js

```diff
@@ -21,3 +21,30 @@ export function formatDate(formatString, date) {
 
   return moment(date || new Date()).format(formatString);
 }
+
+/**
+ * A formatLocaleDate helper to format date with locale using moment js.
+ *
+ * @example
+ *      {{formatDate 'MM/DD/YYYY' 'es' date}}
+ *
+ * @param {string} formatString Based on moment.js.
+ * @param {string} localeString Language or language-country locale string available in https://github.com/moment/moment/tree/develop/locale .
+ * @param {Date} date
+ * @returns {string}
+ */
+export function formatLocaleDate(formatString, localeString, date) {
```

Comment:
Instead of creating a new function `formatLocaleDate`, can you add a third parameter in the existing helper function [`formatDate`](https://github.com/leapfrogtechnology/just-handlebars-helpers/pull/73/files#diff-483dc38c42d387b3d452c409fd049e76R13)? It will make things simple. 

You could make it backward compatible - meaning that it would only apply the locale if third param is not empty otherwise the behavior will be the same as before.

Reply:
Yeah, I can restructure that. Thanks for the reminder. Originally I had some concern before I started testing that I'd need to do extra things to import the other locales on the fly that would be a big expansion of `formatDate` for current users, but that turned out not to be the case.

Reply:
Actually this did seem to be an issue with the node build (though not one I encountered locally), so I just added a twist to pull in the expanded version of moment with locales and that seemed to help (though I'd guess this will bloat `just-handlebars-helpers` a little).